client/me/purchases/purchase-item/index.jsx

```diff
@@ -66,6 +66,27 @@ class PurchaseItem extends Component {
 	getStatus() {
 		const { purchase, translate, locale, moment, name, isJetpack, isDisconnectedSite } = this.props;
 		const expiry = moment( purchase.expiryDate );
+		// To-do: There isn't currently a way to get the taxName based on the country.
```

Comment:
Since this is being used in two places, maybe we can move it to a helper function file such as `client/my-sites/purchases/utils.js` though I suppose this work could be helpful in checkout too, but there isn't a common parent directory where we could nest it 🤔